### PR TITLE
chore: label-schema to include pr title in the footer of the slack message

### PR DESCRIPTION
**Pull request by reversearrow**
### Summary

Update rtBot to include the PR title in the Slack message, instead of displaying 'Powered By rtCamp's GitHub Actions Library'. This change will eliminate the need to visit each commit from the Slack message and will allow us to easily reference the associated pull request.

### Checklist

- [ ] The Pull Request has tests
- [ ] A changelog file has been created under `changelog/unreleased/kong` or `skip-changelog` label added on PR if changelog is unnecessary. [README.md](https://github.com/Kong/gateway-changelog/README.md)
- [ ] There is a user-facing docs PR against https://github.com/Kong/docs.konghq.com - PUT DOCS PR HERE

### Issue reference

Fix #_[issue number]_

### CODE REVIEW

File: .github/workflows/label-schema.yml [hunk 1]
```diff
@@ -11,4 +11,6 @@ jobs:
       uses: rtCamp/action-slack-notify@v2
       continue-on-error: true
       env:
+        PR_TITLE: ${{ github.event.pull_request.title }}
         SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_SCHEMA_CHANGE }}
+        SLACK_FOOTER: $PR_TITLE
```
Comment (flrgh) [inline .github/workflows/label-schema.yml:16]: Correct me if I am wrong, but at a glance, I'm skeptical that `$PR_TITLE` will be propagated to `SLACK_FOOTER`. I would have expected to see something more like...

```suggestion
        SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_SCHEMA_CHANGE }}
        SLACK_FOOTER: ${{ github.event.pull_request.title }}
```

Has this been tested?

Comment (reversearrow) [reply .github/workflows/label-schema.yml:16]: Thanks for the review @flrgh. The suggested approach exposes a command injection threat https://stackoverflow.com/a/76679447. I will test this out and let you know my findings. 

https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions

Comment (flrgh) [reply .github/workflows/label-schema.yml:16]: I don't think there is a command injection, because this action isn't invoking any command.

Comment (reversearrow) [reply .github/workflows/label-schema.yml]: Makes sense, I have updated the PR.

**Review by flrgh — APPROVED** [review]

**Review by zhongweiy — APPROVED** [review]

**Pull request merged**